docs/content/docs/build/containers.md

```diff
@@ -111,7 +114,11 @@ data is shared with other clients. This is an ideal time to create initial data
 
 ### connected/disconnected
 
-The container exposes the connected state of the client and emits connected and disconnected events to notify the caller if the underlying connection is disrupted. Fluid will by default attempt to reconnect in case of lost/intermittent connectivity. However, `connect()` and `disconnect()` can be used to manually control the connection policy. Note about this code:
+The container exposes the connected state of the client and emits connected and disconnected events to notify the caller if the underlying connection is disrupted.
+Fluid will by default attempt to reconnect in case of lost/intermittent connectivity.
+However, `connect()` and `disconnect()` can be used to manually control the connection policy.
```

Comment:
This is interesting topic on general container's observability. Also, this info may be out of date. Even without considering errors, dirty/saved state, I believe we have additional changes around connected state that came in. Please include them here, if so.